automl/vision_object_detection_predict.go

```diff
@@ -0,0 +1,90 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package automl contains samples for Google Cloud AutoML API v1beta1.
+package automl
+
+// [START automl_vision_object_detection_predict]
+import (
+	"context"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+
+	automl "cloud.google.com/go/automl/apiv1beta1"
+	automlpb "google.golang.org/genproto/googleapis/cloud/automl/v1beta1"
+)
+
+// visionObjectDetectionPredict does a prediction for image classification.
+func visionObjectDetectionPredict(w io.Writer, projectID string, location string, modelID string, filePath string) error {
+	// projectID := "my-project-id"
+	// location := "us-central1"
+	// modelID := "IOD123456789..."
+	// file_path := "testdata/salad.jpg"
```

Comment:
Do we need to check in the binary files? I'd prefer to leave them out and download them as-needed form a bucket for tests.

Reply:
This was going to be used for tests anyways so I'll remove them for now